# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/PBrowser.ipdl
# Commit: 50aa16aa36fb
# Full Hash: 50aa16aa36fb3b808d4116377db79bdc6faaa6fe
# Author: Nika Layzell <nika@thelayzells.com>
# Date: 2020-04-08 03:36:50
# Regressor Bug: 1616353
# File Overlap Count: 1
# Description:
#   Bug 1616353 - Part 6.3: Replace many nsILoadContext mutators with assertions, r=smaug
#   
#   Many of these setters are redundant, and duplicate work done elsewhere. As it is
#   no longer correct to set nsILoadContext attributes at these times, they're being
#   replaced with assertions.
# ==============================================================================

diff -r c8c7d1ab9fb4 -r 50aa16aa36fb dom/ipc/PBrowser.ipdl
--- a/dom/ipc/PBrowser.ipdl	Tue Apr 07 21:39:14 2020 +0000
+++ b/dom/ipc/PBrowser.ipdl	Tue Apr 07 21:39:16 2020 +0000
@@ -1014,12 +1014,6 @@
     async AllowScriptsToClose();
 
     /**
-     * Tell the BrowserChild what OriginAttributes it should inherit from. This must
-     * be called before the first non-blank document is loaded in the BrowserChild.
-     */
-    async SetOriginAttributes(OriginAttributes aOriginAttributes);
-
-    /**
      * Pass the current handle for the current native widget to the content
      * process, so it can be used by PuppetWidget.
      */